sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -856,6 +856,17 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
     sql("DROP TABLE alter1")
   }
 
+  test("SPARK-12868 ADD JAR FROM HDFS") {
+    val testJar = "hdfs://nn:8020/foo.jar"
+    // This should fail with unknown host, as its just testing the URL parsing
+    // before SPARK-12868 it was failing with Malformed URI
+    val e = intercept[RuntimeException] {
```

Comment:
I think this test should be improved before merging this. Looking for a RuntimeException to validate that the Jar was registered is brittle and can easily pass when the registration doesn't actually work.